tools/tasks/seed/app.through.ts

```diff
@@ -0,0 +1 @@
+import '../../../src/client/app/main';
```

Comment:
There's no -p option with ng-xi18n yet but it's being discussed on the angular repo ([2201](https://github.com/angular/angular-cli/issues/2201)).